### ISSUE TITLE
Difference between '--word_blank' and '--word_dropout' for DAE 

### ISSUE THREAD

**Issue by bakszero** [facebookresearch/XLM#185]
Hi,

I have a small doubt - what is the difference between the hyperparameters '--word_blank' and '--word_dropout' while constructing the noised version of the input during training? Aren't they both practically the same?

Thanks!

**Comment by glample** [comment]
Hi,

Word blank randomly replaces some words with a <BLANK> token, so the model knows that there was a word here, but it does not know what it is. Word dropout removes some words entirely. So with word blank, you may have:
"I like cats" -> "I <BLANK> cats"
while with word dropout you may have:
"I like cats" -> "I cats"

So the word dropout is a bit more difficult, but overall the results are very similar with either of these words.

**Comment by bakszero** [comment]
Perfect thanks!
